docs/sunpy_stability.yaml

```diff
@@ -0,0 +1,60 @@
+cm:
+  status: dev
+  comments: Stuff
+coordinates:
+  status: dev
+  comments: Stuff
+data:
+  status: dev
+  comments: Stuff
+database:
+  status: dev
+  comments: Stuff
+extern:
+  status: dev
+  comments: Stuff
+image:
+  status: dev
+  comments: Stuff
+instr:
+  status: dev
+  comments: Stuff
+io:
+  status: dev
+  comments: Stuff
+lightcurve:
+  status: dev
+  comments: Stuff
+map:
+  status: dev
+  comments: Stuff
+net:
+  status: dev
```

Comment:
stable, we tend to maintain backwards compat? Certainly after JSOC lands this will be true.